sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/protocol/datalake_rest_client.hpp

```diff
@@ -207,22 +207,19 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
     constexpr static const char* HeaderRequestIsServerEncrypted = "x-ms-request-server-encrypted";
 
     // The value must be "filesystem" for all filesystem operations.
-    class FileSystemResourceType {
+    class FileSystemResource {
```

Comment:
Is this considered private/implementation as it is from the protocol layer?

We need changelog if not

Reply:
It is private